common/beerocks/bcl/source/network/network_utils.cpp

```diff
@@ -533,6 +533,18 @@ uint32_t network_utils::linux_get_iface_index(const std::string &iface_name)
     return iface_index;
 }
 
+std::string network_utils::linux_get_iface_name(uint32_t iface_index)
+{
+    char iface_name[IF_NAMESIZE] = {0};
+    if (!if_indextoname(iface_index, iface_name)) {
```

Comment:
There are currently 6 open-coded calls to if_indextoname, it would be good to replace those as well. In a separate PR, of course.

Reply:
Done: #1461